### DISCUSSION TITLE
CSS , PRO Пользовательский CSS , Пользовательский дизайн

### DISCUSSION THREAD

**Discussion by AlexeiAlexeikov** [totumonline/totum-ru-issues-and-discussions#561] [Как это сделать?] [answered]
Здравствуйте, нужна помощь насчет "PRO Пользовательский CSS". Мне необходимо сделать пользовательский дизайн, включающийся при переключении на темный режим (только изменение цвета разных элементов, без изменения шрифта и т.п.). Сам дизайн достаточно светлый и не сочетается с элементами дефектной темной темы. Я использую CSS от разработчиков для темной темы, которую они прикладывают в обучающем разделе. Есть элементы, которые я не могу найти как изменить их цвет в этом CSS и они совершенно не сочитаются. Например кнопки "добавить" и "редактировать"(у других они могут по другому называться но все равно эти функциональные кнопки присутствуют). Также некоторые ячейки в модальных окнах окрашены в цветы темной темы(коричневым).

Вопросы: 

1) есть ли нормальный прокомментированный CSS который мы можем сами нормально настроить, вставив в "PRO Пользовательский CSS" в настройках, где прописано изменение цвета каждого элемента?

2) данная проблема наводит на идею инвертировать темную тему на светлую(т.е. при переключении на темную у нас та же светлая) и вставлять в "PRO Пользовательский CSS" уже те изменения элементов из CSS для темной темы, которые понятны для чего. Что нам это даст? Элементы которые подтягивались из дефектной темной темы будут подтягиваться из дефолтной светлой и хоть как-то будут сочетаться с пользовательским CSS, который в моем случае в светлом дизайне исполнен. Подытоживая, как это инвертировать?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. Нету.
2. Не знаю чем вам помочь.
3. Когда мы вносим изменения в систему (у v5 изменена например как светлая так и темная) мы не отслеживаем обратную совместимость.

**Reply by ttm-alexeypolunin** [reply]
Я вообще немного решил подумать над организацией например кастомных дизайнов. Это можно делать в рамках платформы имея отдельного человека на фулл-тайм, который занимается фронтом. Какая при этом должна быть команда — не менее 5 человек, тк если 3-4 например, то там не будет фронтовика. 

Какой бюджет у 5 человек = берем 250К/мес + налоги = 370К * 5 = 1870К + общие налоги = 2М в месяц. 

Итак, просим 150 р. за пользователя = 13К платных пользователей. 

Будет это в ближайшее время — я думаю нет. Будет чего угодно придумывать, по 20 человек от одного аккаунта будут работать, лишь бы не платить.

Поэтому мне пишут периодически про эту историю с дизайнами — но мы ей не будем заниматься реально.

**Reply by AlexeiAlexeikov** [reply]
Благодарю, за ответ, поняли вас

**Comment by Re-Requeste** [comment]
![1234](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/e0691c5b-c0f5-4e43-a22f-3a67156a8719)
Еще конечно в процессе разработки, но если мне нужно изменить код css, через браузер ищу элемент который мне нужен, нахожу класс, иду в css и уже там штрудирую... Ну как то так, это максимально быстрый вариант

**Reply by AlexeiAlexeikov** [reply]
А код css вставляете в PRO Пользовательский CSS в настройках ?

**Reply by Re-Requeste** [reply]
К сожалению нет, мне так же нужно изменить дерево и хедер, из-за этого мне проще менять непосредственно **main.css** + я отключил смену с темного на светлое. Ну как отключил, я просто вырезал эту возможность на уровне кода
